src/nxt_clone.h

```diff
@@ -0,0 +1,34 @@
+#ifndef _NXT_CLONE_INCLUDED_
+#define _NXT_CLONE_INCLUDED_
+
+#ifdef NXT_LINUX
+
+#endif
+
+/**
+ * Limit values obtained from `man user_namespaces`
+ */
+#ifdef NXT_LINUX
+
+#include <linux/version.h>
+
+#if LINUX_VERSION_CODE >= KERNEL_VERSION(4,15,0)
+#define NXT_CLONE_MAX_UID_LINES 340
```

Comment:
> What does happen when we try to write more lines than kernel allows?
> 
Write returns EINVAL.

> I'm asking because this type of checks usually doesn't work well. It's better to avoid checking kernel version on compile time. Sometimes users update kernel without recompiling software, and sometimes they use kernels other than those come with the distribution.

I agree with you. I can avoid the checking and just check the write error and in case of `EINVAL` I can add an alert log reminding the user about the limits. (`5` for kernel < 4.15 and `340` otherwise.)
We can return this as an error description in the API in the future as well.